src/layer/S3SLSTRLayer.ts

```diff
@@ -13,7 +13,7 @@ type S3SLSTRFindTilesDatasetParameters = {
 
 export class S3SLSTRLayer extends AbstractSentinelHubV3Layer {
   public readonly dataset = DATASET_S3SLSTR;
-  protected maxCloudCoverPercent: number;
+  public maxCloudCoverPercent: number;
```

Comment:
Should the `orbitDirection` and `view` also be public?

Reply:
Already done, resolving.

Reply:
Thanks, fixed both here and in S-5.